android/java/org/chromium/chrome/browser/crypto_wallet/fragments/dapps/AddSwitchChainNetworkFragment.java

```diff
@@ -50,6 +59,10 @@ public class AddSwitchChainNetworkFragment extends BaseDAppsFragment {
     private boolean hasMultipleAddSwitchChainRequest;
     private List<TwoLineItemDataSource> networks;
     private List<TwoLineItemDataSource> details;
+    private ImageView mFavicon;
+    private FaviconHelper mFaviconHelper;
+    private DefaultFaviconHelper mDefaultFaviconHelper;
+    private BraveWalletService mBraveWalletService;
```

Comment:
that is unused

Reply:
> 

I will start using domain layer from next ticket, I just wanted to develop a little more understanding around it.